aioxmpp/pubsub/service.py

```diff
@@ -537,7 +537,8 @@ def publish(self, jid, node, payload, *, id_=None):
         the node, it is replaced. If no ID is given, a ID is generated by the
         server.
 
-        Return the ID of the item as published.
+        Return the ID of the item as published (or :data:`None` if the server
+        does not inform us).
```

Comment:
While you’re touching this docstring, can you please update it to use ``:param:`` and ``:return:`` and so on in addition to the explanatory text?